spyder/plugins/application/plugin.py

```diff
@@ -121,6 +126,25 @@ def on_mainwindow_visible(self):
             container.give_updates_feedback = False
             container.check_updates(startup=True)
 
+    def update_debug_logs(self):
```

Comment:
Please mode this method to the container. The standard we decided to settle in for all plugins is to declare actions, menus and other graphical things in the container and only leave in the plugin a small interface to be called from others.